lighthouse-core/audits/time-to-interactive.js

```diff
@@ -90,45 +187,23 @@ class TTIMetric extends Audit {
         }
       }
 
-      // Find first 500ms window where Est Input Latency is <50ms at the 90% percentile.
-      let startTime = Math.max(fmpTiming, visuallyReadyTiming) - 50;
-      let endTime;
-      let currentLatency = Infinity;
-      const percentiles = [0.9]; // [0.75, 0.9, 0.99, 1];
-      const threshold = 50;
-      const foundLatencies = [];
-
-      // When we've found a latency that's good enough, we're good.
-      while (currentLatency > threshold) {
-        // While latency is too high, increment just 50ms and look again.
-        startTime += 50;
-        endTime = startTime + 500;
-        // If there's no more room in the trace to look, we're done.
-        if (endTime > endOfTraceTime) {
-          throw new Error('Entire trace was found to be busy.');
-        }
-        // Get our expected latency for the time window
-        const latencies = TracingProcessor.getRiskToResponsiveness(
-          model, trace, startTime, endTime, percentiles);
-        const estLatency = latencies[0].time;
-        foundLatencies.push({
-          estLatency: estLatency,
-          startTime: startTime.toFixed(1)
-        });
+      const times = {fmpTiming, visuallyReadyTiming, endOfTraceTime};
```

Comment:
can you put a `lastTraceEvtTs` into trace-of-tab?

```js
const traceEndTsInMS = trace.traceEvents.reduce(
          (acc, traceEvent) => Math.max(acc, traceEvent.ts), 0) / 1000;
```
it's from will's https://github.com/GoogleChrome/lighthouse/pull/1936/files#diff-fd99fc8068c6555d1fb3d1c70df7c29cR82

wec can then reuse here (and maybe drop the traceviewer dep) and also in will's #1936